### PR TITLE
dthiesse setup updates

### PR DESCRIPTION
**Pull request by dthiesse**
MacOs document ready to go.

### CODE REVIEW

File: setup/set_up_mac.md [hunk 1]
```diff
@@ -53,64 +54,59 @@ You should see a list of usage options.
...
-`$ brew tap cloudfoundry/tap`
-2. Install:  
-`$ brew install cf-cli`
-3. In order to verify the installation, type  
-`cf �v`  
-The response shows the version number.
+### Step 11: Install the IBM Container Service plugins
+There are two plugins associated with the IBM Cloud Container Service that you will need for the labs. They are the plugin for the container service itself (`bx cs` commands) and the plugin for the container registry (`bx cr` commands).
+1. Type  
+`bx plugin install container-service -r Bluemix`
```
Comment (palistra) [inline setup/set_up_mac.md]: I think lines 102 and 103 should be separate commands so might need a `.br` or something?

File: setup/set_up_mac.md [hunk 2]
```diff
@@ -3,23 +3,24 @@ This document provides the steps for installing the software required to begin t
 1. Install Xcode  
 2. Install Homebrew  
 3. Install Java 1.8
```
Comment (palistra) [inline setup/set_up_mac.md:5]: Not sure why Java is needed, but then again I don't know Mac platform?

File: setup/set_up_mac.md [hunk 3]
```diff
@@ -3,23 +3,24 @@ This document provides the steps for installing the software required to begin t
...
-9. Install docker  
-10. Install cloudfoundry CLI  
-11. Install the IBM Container plugin for cloudfoundry  
+4. Create JAVA_HOME environment variable
+5. Install Gradle  
+6. Install GIT  
+7. Install curl  
+8. Install node.js   
+9. Install Docker  
+10. Install Bluemix (IBM Cloud) CLI
```
Comment (palistra) [inline setup/set_up_mac.md:12]: Can we call it the `IBM Cloud (Bluemix)` CLI (giving IBM Cloud higher billing)?

**Review by palistra — COMMENTED** [review]
I added some 'inline' comments, can you see them?  Let me know either way.

File: setup/set_up_mac.md (hunk 1 above)
Comment (dthiesse) [reply setup/set_up_mac.md]: OK. Will do. Sometimes or always you seem to need a <br> between 2 commands.

File: setup/set_up_mac.md (hunk 2 above)
Comment (dthiesse) [reply setup/set_up_mac.md:5]: You need to compile 2 Java apps for the Microservice lab. I think you need Java for that, not just Gradle.

File: setup/set_up_mac.md (hunk 3 above)
Comment (dthiesse) [reply setup/set_up_mac.md:12]: Yes. Will change that.

File: setup/set_up_mac.md (hunk 2 above)
Comment (palistra) [reply setup/set_up_mac.md:5]: Ah!

**Pull request merged**